layouts/partials/footer_scripts.html

```diff
@@ -0,0 +1,72 @@
+{{ if not .Site.Params.disablefancyfonts }}
```

Comment:
This is copied over from the theme, as described [here](https://bwaycer.github.io/hugo_tutorial.hugo/themes/customizing/)